Framework/Algorithms/test/PDCalibrationTest.h

```diff
@@ -475,6 +477,104 @@ class PDCalibrationTest : public CxxTest::TestSuite {
+    TS_ASSERT_THROWS_NOTHING(wsalg.execute());
+    TS_ASSERT(wsalg.isExecuted());
+
+    const std::string params = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n"
+                               "<parameter-file instrument=\"basic_rect\" valid-from=\"1900-01-31T23:59:59\">\n"
+                               "<component-link name=\"basic_rect\">\n"
+                               "<parameter name=\"IkedaCarpenterPV:Alpha0\" type=\"fitting\" >\n"
+                               "       <formula eq=\"1.6\" result-unit=\"TOF\"/>\n"
+                               "        <fixed />\n"
+                               "</parameter>\n"
+                               "<parameter name=\"IkedaCarpenterPV:Alpha1\" type=\"fitting\" >\n"
+                               "        <formula eq=\"1.5\" result-unit=\"TOF\"/>\n"
+                               "        <fixed />\n"
+                               "</parameter>\n"
+                               "<parameter name=\"IkedaCarpenterPV:Beta0\" type=\"fitting\" >\n"
+                               "        <formula eq=\"31.7927\" result-unit=\"TOF\"/>\n"
+                               "        <fixed />\n"
+                               "</parameter>\n"
+                               "<parameter name=\"IkedaCarpenterPV:Kappa\" type=\"fitting\" >\n"
+                               "        <formula eq=\"0.1\"/>\n"
+                               "        <fixed />\n"
+                               "</parameter>\n"
+                               "    <parameter name=\"IkedaCarpenterPV:SigmaSquared\" type=\"fitting\">\n"
+                               "        <formula eq=\"1.0\" unit=\"dSpacing\" result-unit=\"TOF^2\"/>\n"
+                               "    </parameter>\n"
+                               "    <parameter name=\"IkedaCarpenterPV:Gamma\" type=\"fitting\">\n"
+                               "        <formula eq=\"1.0\" unit=\"dSpacing\" result-unit=\"TOF\"/>\n"
+                               "    </parameter>\n"
+                               "</component-link>\n"
+                               "</parameter-file>";
+
+    LoadParameterFile paramalg;
+    TS_ASSERT_THROWS_NOTHING(paramalg.initialize());
+    TS_ASSERT(paramalg.isInitialized());
+    TS_ASSERT_THROWS_NOTHING(paramalg.setPropertyValue("Workspace", "ws"));
+    TS_ASSERT_THROWS_NOTHING(paramalg.setPropertyValue("ParameterXML", params));
+    TS_ASSERT_THROWS_NOTHING(paramalg.execute());
+    TS_ASSERT(paramalg.isExecuted());
+
+    MatrixWorkspace_const_sptr ws = AnalysisDataService::Instance().retrieveWS<MatrixWorkspace>("ws");
+    TS_ASSERT(ws);
+
+    MoveInstrumentComponent movealg;
+    TS_ASSERT_THROWS_NOTHING(movealg.initialize());
+    TS_ASSERT(movealg.isInitialized());
+    TS_ASSERT_THROWS_NOTHING(movealg.setProperty("Workspace", "ws"));
+    TS_ASSERT_THROWS_NOTHING(movealg.setPropertyValue("ComponentName", "bank1"));
+    TS_ASSERT_THROWS_NOTHING(movealg.setProperty("X", 1.0));
```

Comment:
So this is the already perfectly calibrated positions that directly corresponds to `difc = 2208.287616521762`, can we change this so X=1.01 and Z=1.01, that will give a starting difc of 2211.02 and we check that is still get the correct original difc.

Reply:
It is now giving a DIFC of around 2208.31 with the new instrument shift which seems to be good